pandas/core/ops.py

```diff
@@ -543,9 +543,9 @@ def _offset(lvalues, rvalues):
 
                 # with tz, convert to UTC
                 if self.is_datetime64tz_lhs:
-                    lvalues = lvalues.tz_localize(None)
```

Comment:
I *think* this localization can be removed, as adding timedelta is unrelated to the timezone.